lib/core/src/display/shape/text/text_field.rs

```diff
@@ -185,19 +185,51 @@ shared! { TextField
 }
 
 
+// === Constructor ===
+
+impl TextField {
+    /// Create new TextField.
+    pub fn new(world:&World, initial_content:&str, properties:TextFieldProperties) -> Self {
```

Comment:
I don't like the idea to put initial content to constructor. Constructors should be as minimal as possible. So it would be amazing if it will consume only world. Everything else should be "setupable" after the creation.

Reply:
Constructors should take all the data to make the struct working and usable. For initial content - perhaps it can be removed, but for me its nice to give a programmer an option "You can create object with content!", and there is no problem with giving default if you don't care.
As for properies: everyone will set it right after construction, because they are required to display TextField, so it should be as constructor argument. Again, the `default` function comes to help.